.travis.yml

```diff
@@ -133,6 +133,9 @@ matrix:
           - avsm
           packages: *coqide-packages
 
+before_install:
+- if [ ! "${TRAVIS_PULL_REQUEST}" == "false" ]; then git fetch origin +${TRAVIS_BRANCH} && git merge FETCH_HEAD --no-edit; fi
```

Comment:
Because I am a noob in bash. But yes, it must be valid because I've been testing this branch on my fork (even creating fake PRs to test it properly) and it worked. Anyways, I've updated according to your suggestion.